kpconv_torch/utils/config.py

```diff
@@ -227,12 +236,18 @@ def __init__(self):
             if "global" in block or "upsample" in block:
                 break
 
-    def load(self, path):
+    def set_chosen_log_folder(self, chosen_log_folder):
+        self.chosen_log_folder = chosen_log_folder
+    
+    def set_output_folder(self, output_folder):
+        self.output_folder = output_folder
```

Comment:
Do we really need these functions? These are single-line statements, they may be used "as is" in the corresponding modules.

Reply:
Removed.